README.md

```diff
@@ -37,8 +37,24 @@ To check whether any format violation exists, run `spotlessCheck` gradle task. I
 
 Both plugins are enabled on gradle's `compileJava` task.
 
-## References
+### Testing
```

Comment:
We could give the CLI command to run all the tests, somewhere in this section

Reply:
good idea. I added gradle command usage examples for both tests.